### ISSUE TITLE
DietPi issuse

### ISSUE THREAD

**Issue by darioristic** [MichaIng/DietPi#1668]
#### Required Information:
- DietPi Version | v6.3
- SBC Device     | RPi 3 Model B (armv7l) (index=3)
- Distro         | stretch (index=4)
- Command        | Connection test: https://www.mirrorservice.org/sites/archive.raspbian.org/raspbian
- Error Handler  | G_ERROR_HANDLER

#### Additional Information (if applicable):
- Software title | DietPi-Software

#### Additional logs:
```
Log file contents:
Spider mode enabled. Check if remote file exists.
--2018-03-31 11:55:07--  https://www.mirrorservice.org/sites/archive.raspbian.org/raspbian
Resolving www.mirrorservice.org (www.mirrorservice.org)... failed: Connection timed out.
wget: unable to resolve host address ‘www.mirrorservice.org’
```

**Comment by Fourdee** [comment]
@darioristic 

Hi,

Appears to be online here. If the problem still persist, please can you answer the following:
- Is PiHole installed?
- Paste results of ```route```
- DHCP or STATIC IP? Check in ```dietpi-config``` > Networking Options: Adapters

**Comment by ghost** [comment]
@darioristic @Fourdee Same procedure as everytime - ppl should read your fix upon this problem... it's not the PiHole stuff - it is the problem of the changed repo in the sources.list.

-> nano /etc/apt/sources.list and change the entry to http://raspbian.raspberrypi.org/raspbian #1659

**Comment by MichaIng** [comment]
@SuBLiNeR 
Not in this case, the mirror URL is still correct, seemed to be a temporary unavailable `www.mirrorservice.org` or indeed some network configuration issue.

But to avoid any future issue with this, it is no bad idea to anyway adjust `/etc/apt/sources.list` accordingly, if already touching the system 🙂.

@darioristic 
Just for reference, what we talk about: https://github.com/Fourdee/DietPi/issues/1659

**Comment by ghost** [comment]
@MichaIng 
But this is how i solved this even before #1659 was than marked as being temp solved. Nah, the mirrorservice.org is always a bottleneck since the past.... :1st_place_medal:

**Comment by Fourdee** [comment]
No response, marking as closed. Please reopen if required.
